manager/notifier.go

```diff
@@ -276,6 +278,46 @@ func (n *notifier) sendPushoverNotification(token, userKey string, a *Alert) err
 	return err
 }
 
+func (n *notifier) sendHipChatNotification(token, room string, a *Alert) error {
+	// https://www.hipchat.com/
+        incidentKey := a.Fingerprint()
+        buf, err := json.Marshal(map[string]interface{}{
```

Comment:
@brian-brazil Not sure what you mean here? It's basically just doing the same as the code for PagerDuty?